test/dev/basic.generate.test.js

```diff
@@ -36,6 +36,13 @@ describe('basic generate', () => {
     nuxt.hook('generate:done', () => {
       writeFileSync(changedFileName, '')
     })
+    nuxt.hook('export:page', ({ page, errors }) => {
+      if (errors.length > 0) {
+        if (page.path.indexOf('/skip-on-fail') === 0) {
```

Comment:
Refactor tips:
 - `> 0` can be omitted
- `indexOf === 0` can be `includes` (or simply eqeqeq)
- Both conditions can be combined